src/docs/sdk/event-payloads/contexts.mdx

```diff
@@ -137,6 +145,10 @@ The `type` and default key is `"device"`.
 
 : _Optional_. The timezone of the device. For example, `Europe/Vienna`.
 
+`language`
```

Comment:
See https://github.com/getsentry/sentry/blob/9707fd3f98fa51d5d363cbe34b59aa51588db246/static/app/components/events/contexts/device/getDeviceKnownDataDetails.tsx#L182

Reply:
Nice! Do we have examples of SDKs that send it in this format or another already?

Reply:
Android does it. The format is for example en_US.

Reply:
Feel free to raise an issue on getsentry/sentry requesting that. 

I requested one feature once and it landed super quickly: https://github.com/getsentry/sentry/issues/24209